### PR TITLE
Added updated plugin version of Fan Speed Slider Plugin

### PR DESCRIPTION
**Pull request by mival**

### CODE REVIEW

File: _plugins/FanSpeedSlider.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: plugin
+id: fanspeedslider
+title: Fan Speed Control
+description: Adds fan speed control buttons to Control tab -> General
+archive: https://github.com/mival/OctoPrint-FanSpeedSlider/archive/main.zip
+homepage: https://github.com/mival/OctoPrint-FanSpeedSlider
+source: https://github.com/mival/OctoPrint-FanSpeedSlider
+author: mival
```
Comment (jneilliii) [inline _plugins/FanSpeedSlider.md]: would be nice to credit all the authors here as an array. change `author: mival` to
```
authors:
- mival
- jneilliii
- czM1K3
- ntoff
```

You also need to update the software update check line [here](https://github.com/mival/OctoPrint-FanSpeedSlider/blob/704c37d853b86f7b5241ba4748217104fd4f7422/octoprint_fanspeedslider/__init__.py#L141) to your github username.

**Review by jneilliii — CHANGES_REQUESTED** [review]
should probably include all contributors to authors since this is a fork of a fork and you need to rename the file `_plugins/FanSpeedSlider.md` to `plugins/fanspeedslider.md` to match the plugin_identifier.

File: _plugins/FanSpeedSlider.md (hunk 1 above)
Comment (mival) [reply _plugins/FanSpeedSlider.md]: @jneilliii thank you for review, I updated code and plugin info

**Review by jneilliii — APPROVED** [review]

File: _plugins/fanspeedslider.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: plugin
+id: fanspeedslider
+title: Fan Speed Control
+description: Adds fan speed control buttons to Control tab -> General
+archive: https://github.com/mival/OctoPrint-FanSpeedSlider/archive/main.zip
```
Comment (jneilliii) [inline _plugins/fanspeedslider.md:6]: technically should be 
```
archive: https://github.com/mival/OctoPrint-FanSpeedSlider/archive/master.zip
```
to match the branch name.

**Review by jneilliii — CHANGES_REQUESTED** [review]
sorry I missed this before @mival but update this URL and one last thing on the plugin's repo. You need to create a release tagged 0.2.3 to match the plugin version identified in setup.py.

File: _plugins/fanspeedslider.md (hunk 2 above)
Comment (mival) [reply _plugins/fanspeedslider.md:6]: @jneilliii no problem, but rather I renamed master -> main to keep with new naming convention. Hopefully I also renamed all occurrences in repo.

**Comment by mival** [comment]
@jneilliii and hopefully I created tag for version and also release for that tag.

**Review by jneilliii — APPROVED** [review]

**Pull request merged**